### DISCUSSION TITLE
请问有办法关掉固定的顶部和底部弹幕吗

### DISCUSSION THREAD

**Discussion by Darkvanci** [Tony15246/uosc_danmaku#72] [Q&A] [answered]
您好 请问有办法关掉固定的顶部和底部弹幕吗

**Comment by Darkvanci** [comment]
还有一个问题 能不能默认打开弹幕，每次开始已经自动加载好弹幕了，但是要按J显示，切换到下一集也要重新打开

**Reply by Tony15246** [reply]
按理来说弹幕的开关状态是记忆上一次播放的状态，上一个视频中弹幕是打开的下一个视频弹幕就是打开的，上一个视频中是关闭的下一个视频就是关闭的。这种情况不是插件的正常行为，可能是bug

可以提供一下出现这种问题时的具体信息吗？你使用的是什么操作系统？你是使用的代码的版本是github最新一次提交版的还是说是最新一次release版的？你播放的是本地视频文件还是网络视频？出现这个问题的时候`uosc_danmaku.conf`配置文件中配置项的配置是什么样的？

**Reply by Darkvanci** [reply]
1.win10系统，使用的包是MPV_lazy
2.版本是最新的release版，播放本地文件
3.没有创建uosc文件，我用的是option更改，内容如下
![图片](https://github.com/user-attachments/assets/2ce5a716-4820-4e1e-8ebc-ce3e6678571c)

**Reply by Tony15246** [reply, marked as answer]
release v1.1.2版落后了最新版源代码不少了，问题在release v1.2.0中可能不存在了。你先把插件代码从最新release版替换成release v1.2.0试试看弹幕一直关闭的问题是否还存在。如果替换完后该问题还存在的话，就把mpv的运行日志发给我看一下，我找找问题到底出在哪。

**Reply by Tony15246** [reply]
对了，包括你提到的这个问题

> 您好 请问有办法关掉固定的顶部和底部弹幕吗

这个按类型屏蔽功能也是在release v1.2.0里才有

**Reply by Darkvanci** [reply]
换新版本解决问题了 非常感谢

**Comment by Tony15246** [comment]
> 您好 请问有办法关掉固定的顶部和底部弹幕吗

想要关掉关掉固定的顶部和底部弹幕在script-opts中创建uosc_danmaku.conf文件并添加如下内容

```
blockmode=TOP-BOTTOM
```

blockmode用于指定不会显示在屏幕上的弹幕类型。使用“-”连接类型名称，例如“L2R-TOP-BOTTOM”。可用的类型包括：L2R,R2L,TOP,BOTTOM,SPECIAL,COLOR,REPEAT
